using/product/reputation.md

```diff
@@ -65,21 +65,22 @@ By default, `12` reputation is awarded when a pull request is merged that was op
+| --------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `x:rep/tiny`    | 3 (~5 mins)   | <ul><li>Fixing a single typo or link</li><li>Removing a blank line or adding a line break</li><li>Changing/adding a single code comment</li></ul>                                                                                                                                                                                                                   |
+| `x:rep/small`   | 5 (~10 mins)  | <ul><li>Fixing a single test case, task or example</li><li>Fixing multiple typos or links in a single file</li><li>Clarifying content by adding a few lines to a file</li></ul>                                                                                                                                                                                     |
+| `x:rep/medium`  | 12 (~30 mins) | <ul><li>Syncing an exercise with problem-specifications (incl. edits)</li><li>Adding one or more test cases from scratch</li><li>Improving multiple files in an exercise</li><li>Adding mentor notes for an exercise from scratch</li><li>Fixing a small bug in a test runner/analyzer/representer</li><li>Adding analyzer comments for a single exericse</li></ul> |
+| `x:rep/large`   | 30 (~2 hrs)   | <ul><li>Adding a new concept or practice exercise</li><li>Adding new concept documentation</li><li>Substantial re-writing of an existing concept or exercise</li><li>Adding new CI scripts or other automation</li></ul>                                                                                                                                            |
+| `x:rep/massive` | 100 (~5 hrs)  | <ul><li>Creating a test-runner, analyzer, representer or generator from scratch</li><li>Major refactors to those tools</li><li>Creating major documentation from scratch (e.g. contribution or testing guides)</li></ul>                                                                                                                                            |
 
 The examples above can serve as rough orientation when to apply which label but maintainers are free to use their own judgement.
 
+- The estimated number of time spent should be interpreted as the average time a _maintainer_ would spend on doing the PR.
 - If more than one label is specified, the label with the highest reputation value determines the awarded reputation.
 - If a pull request is still open, no reputation is awarded (yet).
 - If a pull request is closed _without_ merging, no reputation is awarded.
```

Comment:
This should be "exceptionable".  There can be value in the discussion of code that does not come in, and a historical record of why.  So reputation should be able to be given even if a PR is not accepted.

Reply:
We have others ways of rewarding reputation for those cases.